_components/modal/guidance/implementation.md

```diff
@@ -1,6 +1,6 @@
 - **Use unique ids.** Each `.usa-modal` must have a unique id so that openers can associate them with their `aria-controls` attribute.
 
-- **Openers.** A single modal can have multiple openers. Each opener requires `data-open-modal` and `aria-controls=”MODAL_ID”` attributes. Openers can be coded either as `<button type="button">` or `<a>`. Using `<a>` helps link to modals in the event javascript fails.
+- **Openers.** A single modal can have multiple openers. Each opener requires `data-open-modal` and `aria-controls=”[modal_id]”` attributes. Openers can be coded as either `<button>` or `<a>` elements.
```

Comment:
> **Note** 
> Removed "Using `<a>` helps link to modals in the event javascript fails" because it doesn't seem like there is a progressive enhancement here. I might have misunderstood the meaning though, so happy to return if it is still relevant.

Reply:
Seems to be relevant because if the javascript fails to load the modal trigger will still redirect users to the modal content.

If it was a button, you can't reference content via `href` and it won't do anything.

Reply:
Thanks. I restored the copy in [a7e3b52](https://github.com/uswds/uswds-site/pull/2162/commits/a7e3b520134d560db907d23e83a247c0c4c9b32d)